install_deps.sh

```diff
@@ -14,22 +14,22 @@ WGET_FLAGS="--https-only"
 echo ca_certificate=/etc/ssl/certs/ca-certificates.crt > ~/.wgetrc
 
 # Install tools
-apt-get update && apt-get ${APT_GET_FLAGS} install \
+sudo apt-get update && sudo apt-get ${APT_GET_FLAGS} install \
     autoconf \
     automake \
     libtool \
     build-essential \
     ccache \
-    clang-9 \
-    clang-format-10 \
-    clang-tidy-10 \
+    clang \
```

Comment:
We are using a specific version of clang. Are you sure when you install clang package it will install the this specific clang version?

Reply:
I will revert this change. I checked that clang version 6.0 was being installed on Ubuntu 18.04. However, these commands will have to be changed once we support Ubuntu 20.04 or higher.